### PR TITLE
19723 add min-height to timelines content section

### PR DESCRIPTION
**Pull request by vcostin**
[`* `#19723` UI-select for choosing timeline report cut off at bottom"`](https://community.openproject.org/work_packages/19723)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://vcostin-bug-19723-choosing-timeline-report-select-cut.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://vcostin-bug-19723-choosing-timeline-report-select-cut.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by NobodysNightmare** [comment]
@myabc Why the WIP label? @vcostin put this thing to "PR created", so I guess this is ready for review...

@vcostin Is it?

File: app/assets/stylesheets/content/_timelines.scss [hunk 1]
```diff
@@ -71,6 +71,10 @@
   }
 }
 
+.controller-timelines #content {
+  min-height: 700px;
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/content/_timelines.scss:76]: This is quite a specific **hack** to allow the select2 menu to be visible under all circumstances.

You really should (I don't want to say "must", but you actually must) add a comment that clearly states the intent of this specific CSS rule.

Something like:

``` CSS
/*HACK: Use a min-height to ensure that the timeline_select select2 is not cropped at the bottom*/
```

Comment (vcostin) [inline app/assets/stylesheets/content/_timelines.scss:76]: :ok_hand:

**Comment by myabc** [comment]
@NobodysNightmare 

> @myabc Why the WIP label? @vcostin put this thing to "PR created", so I guess this is ready for review...

This PR may be redundant once #2910 is merged.

**Comment by NobodysNightmare** [comment]
@myabc Would you agree on merging this PR in the meantime? I am not sure what the ETA on #2910 is...

**Comment by myabc** [comment]
@NobodysNightmare I'm expecting that #2910 will be in a reviewable form this week. Reverting this change will add a nominal amount of work.

File: app/assets/stylesheets/content/_timelines.scss [hunk 2]
```diff
@@ -71,6 +71,11 @@
   }
 }
 
+/*HACK: Use a min-height to ensure that the timeline_select select2 is not cropped at the bottom*/
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/content/_timelines.scss]: Thank you @houndci, didn't know that SCSS allows cool-style comments!

**Comment by NobodysNightmare** [comment]
@myabc Okay, so I will wait with merging this one... But I'd rather not close it until the fix from your branch is finally applied.

edit: That is assuming that your fix does not involve a hack...

**Comment by myabc** [comment]
> edit: That is assuming that your fix does not involve a hack...

the h-word isn't in my vocabulary!

File: app/assets/stylesheets/content/_timelines.scss (hunk 2 above)
Comment (myabc) [inline app/assets/stylesheets/content/_timelines.scss]: @NobodysNightmare better tooling (e.g. code-linting) is compelling reason for moving to SCSS syntax everywhere: https://community.openproject.org/work_packages/18422

**Comment by floriank** [comment]
@myabc - just to clarify: #2910 will include a fix for this issue?

**Pull request merged**